docs/devices/WV704R0A0902.md

```diff
@@ -36,88 +36,33 @@ If central light shows a solid green, your TRV has been paired and is connected
+
+### Radiator Thermostat Recalibration
+
+In order to recalibrate a radiator thermostat, please perform the following;
+1. First, please ensure that the radiator thermostat is firmly screwed onto the radiator.
+2. Then, twist the cap in the '+' direction for 8 seconds until the valve fully opens, and the red light comes on.
+3. Release the cap.
+4. Then please twist the cap in the '-' direction for 2 seconds until the blue light comes on and the valve closes.
+5. This ensures that the radiator thermostat is applying suitable pressure to the radiator pin when it is supposed to be closed.
 
 ### Controlling
 
 Valve sometimes fails to respond/acknowledge a setting, just send the command again.
 
-- Get local temperature in degrees Celsius.
-    ```json
-    {
-        "local_temperature": ""
-    }
-    ```
-
-- Get or set occupied heating setpoint to *&lt;temperature&gt;* in degrees Celsius. Possible values: 7 to 30 by default; leave empty to read.
-    ```json
-    {
-        "occupied_heating_setpoint": "<temperature>"
-    }
-    ```
-
-- Get or set the keypad lock, i.e. enable/disable twist-top boost. Possible values: "unlock", "lock1". **WARNING** Setting the keypad to lock (i.e. "lock1") disables the twist-top boost button which impedes setting the valve to installation mode or hard-resetting it.
-    ```json
-    {
-        "keypad_lockout": ""
-    }
-    ```
-
-- Get the battery level, in percentage. Possible values 0 to 100 (%). The level corresponds voltages in the range 2.2 V (0%) to 3.0 V (100%). The levels where chosen so a 0% indicates that batteries must be replaced, but the valve will still work.
-    ```json
-    {
-        "battery": ""
-    }
-    ```
-
-- Get the battery voltage. Possible values 0.0 to 3.0 (V). This is a direct measure of the battery voltage.
-    ```json
-    {
-        "voltage": ""
-    }
-    ```
-
-- Get the (latest) twist-top boost state. Possible values:
-
-    | Attribute Value | Description |
-    | -----------------|----------------------------------------------- |
-    | 1 | Boost up |
-    | 0 | No boost |
-    | -1 | Boost down |
-
-    Note that the valve will not automatically toggle the value to 0 after some time, i.e. configuration/automation logic is needed to use the value. E.g. in Home Assistant the `expire_after` setting can be used to handle this.
-
-- Get the valve position / heating demand. Will report the valve position or heating amount depending. Possible values 0 to 100 (%).
-    ```json
-    {
-        "pi_heating_demand": ""
-    }
-    ```
-- Get the valve motor state. Possible values: "Stall", others. String description of the motor state. Seems to change when the motor is moving.
-    ```json
-    {
-        "MOT": ""
-    }
-    ```
-- Get the Zigbee link quality. Possible values 0 to 100 (%).
-    ```json
-    {
-        "linkquality": ""
-    }
-    ```
 <!-- Notes END: Do not edit below this line -->
```

Comment:
Please do not make changes below this line, the exposes of this device in the zigbee-herdsman-converters has to be updates instead

Reply:
Fixed.